js/module.3CC.js

```diff
@@ -0,0 +1,51 @@
+const getDailyCalories = day => {
+    day = day.toLowerCase();
+    switch (day) {
+        case 'понедельник':
+            return 5000;
+        case 'вторник':
+            return 1300;
+        case 'среда':
+            return 1200;
+        case 'четверг':
+            return 2500;
+        case 'пятница':
+            return 2500;
+        case 'суббота':
+            return 2500;
+        case 'воскресенье':
+            return 2500;
+    }
+}
+
+const getActualCaloriesQty = () => {
+    getDailyCalories('Понедельник');
```

Comment:
since you already call this method, it is better to write a result into variable, and then use it